packages/admin-e2e-tests/src/utils/actions.ts

```diff
@@ -147,7 +153,7 @@ export const waitForElementByTextWithoutThrow = async (
 		if ( selected ) {
 			break;
 		}
-		await page.waitFor( 1000 );
+		await new Promise( ( resolve ) => setTimeout( resolve, 1000 ) );
```

Comment:
Should we be using `waitForTimeout` here?

Reply:
Implemented in https://github.com/woocommerce/woocommerce-admin/pull/7646/commits/387c31aa44c58a845d2742b0492f35b36077ebe2